src/models/input_ids.cpp

```diff
@@ -5,7 +5,7 @@
 
 namespace Generators {
 
-InputIDs::InputIDs(State& state)
+InputIDsDefault::InputIDsDefault(State& state)
```

Comment:
Do we need to add `Default` to the end of the names? There's already separate class names for these models (e.g. `WindowedInputIDs`).

Reply:
There is `InputIds` (base class), and `InputIdsDefault` and `WindowedInputIds` derived classes.

I initially had `InputIdsInterface` as the base and `InputIds` and `SlidingWindowInputIds` as derived.

Do you have any suggestions on the names?

Reply:
one suggestion/opinion is that `DefaultInputIDs` sounds more consistent with `WindowedInputIDs`.

Reply:
similar suggestion for other names
`PositionIDsDefault` -> `DefaultPositionIDs`
`KeyValueCacheDefault` -> `DefaultKeyValueCache`
`KeyValueCacheDefault_Combined` -> `CombinedKeyValueCache`